acceptance: TestDockerCLI_test_copy failed

acceptance.TestDockerCLI_test_copy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/11069589?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/11069589?buildTab=artifacts#/) on master @ [9db3a0eaf6cf0f669352a09c7798c41b21a3a4bd](https://github.com/cockroachdb/cockroach/commits/9db3a0eaf6cf0f669352a09c7798c41b21a3a4bd):


```
=== RUN   TestDockerCLI_test_copy
    test_log_scope.go:167: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/ba7bd39f117af684bba3ca6bf23662ce/logTestDockerCLI_test_copy1719064348
    test_log_scope.go:81: use -show-logs to present logs inline
=== CONT  TestDockerCLI_test_copy
    generated_cli_test.go:89: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/ba7bd39f117af684bba3ca6bf23662ce/logTestDockerCLI_test_copy1719064348
--- FAIL: TestDockerCLI_test_copy (54.33s)
=== RUN   TestDockerCLI_test_copy/test_copy.tcl
=== CONT  TestDockerCLI_test_copy/test_copy.tcl
    util_docker.go:254: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_copy/test_copy.tcl (42.78s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_copy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-30134

--- a/pkg/cli/interactive_tests/test_copy.tcl
+++ b/pkg/cli/interactive_tests/test_copy.tcl
@@ -4,7 +4,7 @@ source [file join [file dirname $argv0] common.tcl]
 
 start_server $argv
 
-spawn $argv sql 
+spawn $argv sql --no-line-editor
 eexpect root@
 
 send "drop table if exists t;\r"
@@ -122,7 +122,7 @@ end_test
 send_eof
 eexpect eof
 
-spawn $argv sql
+spawn $argv sql --no-line-editor
 eexpect root@
 
 ## The following test can be re-enabled after fixing this issue:

